ui/admin/app/routes/scopes/scope/session-recordings/session-recording/channels-by-connection/channel/index.js

```diff
@@ -24,7 +25,13 @@ export default class ScopesScopeSessionRecordingsSessionRecordingChannelsByConne
       try {
         asciicast = await channelRecording.getAsciicast();
       } catch (e) {
-        // no op
+        // Alert user of error occurred during download.
+        const error = e.errors[0];
```

Comment:
Should we have used the `notifyError` decorator here or does it not make sense?

Reply:
I am thinking that using try catch for the model hook is a better pattern then letting the error propagate since hooks are handled differently from other methods & a model hook does return a value whereas actions (that use the decorator) don't return anything.